smf/switch_zone_setup/switch_zone_setup

```diff
@@ -1,49 +0,0 @@
-#!/bin/ksh
```

Comment:
This script has been replaced by a command in the zone-setup CLI in the `zone-setup/` directory